### PR TITLE
Only iterate through mostRecentRequests if it exists 😋

### LINKED ISSUE #452: Viewing archived plans blows up when mostRecentRequests is undefined 😭
wompwomp

### PR DESCRIPTION
**Pull request by AllenBW**
fixes #452

Now yah can see archived plans, and the whole of v2v doesn't crash 😬

### CODE REVIEW

File: app/javascript/react/screens/App/Overview/components/Migrations/MigrationsCompletedList.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ const MigrationsCompletedList = ({
             const failed = mostRecentRequest && mostRecentRequest.status === 'Error';
 
             const tasks = {};
-            mostRecentRequest.miq_request_tasks.forEach(task => {
+            if(mostRecentRequest) mostRecentRequest.miq_request_tasks.forEach(task => {
```
Comment (AparnaKarve) [inline app/javascript/react/screens/App/Overview/components/Migrations/MigrationsCompletedList.js]: how about -
```js
mostRecentRequest && mostRecentRequest.miq_request_tasks.forEach(task => {
```

like it's done above

Comment (AllenBW) [reply app/javascript/react/screens/App/Overview/components/Migrations/MigrationsCompletedList.js]: intentionally and for a very good reason
screen shot 2018-06-28 at 8 37 13 pm


eslint is not a fan

Comment (AparnaKarve) [reply app/javascript/react/screens/App/Overview/components/Migrations/MigrationsCompletedList.js]: Ok, fair.

**Comment by AparnaKarve** [comment]
@AllenBW Changing this to gaprindashvili/yes, since the bug was introduced by https://github.com/ManageIQ/miq_v2v_ui_plugin/pull/444, which is also gaprindashvili/yes

**Pull request merged**